leasing/models/lease.py

```diff
@@ -1005,7 +1006,7 @@ def determine_payable_rents_and_periods(  # noqa: TODO
                             date_range_start=start_date, date_range_end=end_date
                         ),
                         "last_billing_period": False,
-                        "override_receivable_type": rent.override_receivable_type,
+                        "override_receivable_type": rent.get_override_receivable_type(),
```

Comment:
This feels like an important discussion point, I don't yet understand the implementation here as just started reviewing. However, if it is required to match one override_receivable_type to a rent, perhaps it should be set to a rent instead of contractrent. Selecting the first() value out of many feels wrong. Could it cause unexpected invoices to be generated?

Reply:
Well, the original implementation was in Rent, because the intention is that AKV/KuVa rents only have one override receivable type, but also only one contractrent. We decided to move the property to ContractRent instead, because in the UI it must be handled as part of the contractrent element, instead of general property of the rent. Ohjausryhmä thinks it's part of contractrent instead of rent.